grunt/tasks/npm-react-addons.js

```diff
@@ -6,46 +6,55 @@ var path = require('path');
 
 var addons = {
   CSSTransitionGroup: {
+    package: 'react',
     module: 'ReactCSSTransitionGroup',
     name: 'css-transition-group',
     docs: 'animation',
   },
   LinkedStateMixin: {
+    package: 'react',
     module: 'LinkedStateMixin',
     name: 'linked-state-mixin',
     docs: 'two-way-binding-helpers',
   },
   Perf: {
+    package: 'react-dom',
     module: 'ReactPerf',
     name: 'perf',
     docs: 'perf',
   },
   PureRenderMixin: {
+    package: 'react',
     module: 'ReactComponentWithPureRenderMixin',
     name: 'pure-render-mixin',
     docs: 'pure-render-mixin',
   },
   TestUtils: {
+    package: 'react-dom',
```

Comment:
We need to account for the `peerDependencies` field in package.json. Right now the template has `react` as the dep but we'll want to override with this (or even just stop having it in the template at all and always inject - just need the version info too).